File: Baekjoon_Online_Judge/13_geometry_1/03_boj_4153.cpp
#include <iostream>
#include <algorithm>

bool isRightTriangle(int n1, int n2, int n3){
	int sides[3] = {n1, n2, n3};
	std::sort(sides, sides+3);
	return sides[2]*sides[2] == sides[1]*sides[1] + sides[0]*sides[0] ? true: false;
}

int main() {
	int x, y, z;
	while(true){
		scanf("%d %d %d", &x, &y, &z);
		if(x == 0)
			break;
		isRightTriangle(x, y, z) ? printf("right\n") : printf("wrong\n");
	}
}

File: Baekjoon_Online_Judge/13_geometry_1/03_boj_4153.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

input.pop();
const result = input.map(v => {
	const [max, s1, s2] = v.split(" ").map(v => parseInt(v)).sort((a, b) => b - a);
	return max**2 === s1**2 + s2**2 ? "right" : "wrong";
})
console.log(result.join("\n"));